api/maven-api-model/src/main/java/org/apache/maven/api/model/InputLocation.java

```diff
@@ -83,6 +95,13 @@ public Map<Object, InputLocation> getLocations() {
         return locations;
     }
 
+    /**
+     * Gets the input location that caused this model to be read.
+     */
```

Comment:
we need an `@since` here to remember

Reply:
"that caused this model to be read."?
I need to dig more into code to propose another description, but this one is not clear